src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,75 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { FC } from 'react';
+import { FilterType } from '../../types/FilterType';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+interface Props {
+  value: string,
+  filterTypeValue: FilterType,
+  onQueryChange: (query: string) => void
+  onFilterType: (filterTypeValue: FilterType) => void
+}
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+export const TodoFilter: FC<Props> = (props) => {
+  const {
+    value,
+    filterTypeValue,
+    onFilterType,
+    onQueryChange,
+  } = props;
+
+  return (
+    <form className="field has-addons">
+      <p className="control">
+        <span className="select">
+          <select
+            data-cy="statusSelect"
+            value={filterTypeValue}
+            onChange={({ target }) => {
+              onFilterType(target.value as FilterType);
+            }}
+          >
+            {Object.values(FilterType).map(option => (
+              <option
+                value={option}
+                key={option}
+              >
+                {option.slice(0, 1).toUpperCase() + option.slice(1)}
```

Comment:
GJ, but also you could write enum value with capital value

Reply:
I tried, but tests fail because of this. They are searching for values in lower case:(